packages/assets-controllers/src/Standards/NftStandards/ERC1155/ERC1155Standard.ts

```diff
@@ -92,18 +84,9 @@ export class ERC1155Standard {
     contractAddress: string,
     address: string,
     tokenId: string,
-  ): Promise<number> => {
-    const contract = this.web3.eth.contract(abiERC1155).at(contractAddress);
-    return new Promise<number>((resolve, reject) => {
-      contract.balanceOf(address, tokenId, (error: Error, result: number) => {
-        /* istanbul ignore if */
-        if (error) {
-          reject(error);
-          return;
-        }
-        resolve(result);
-      });
-    });
+  ): Promise<BN> => {
```

Comment:
Huh. I'm guessing this was changed because `number` can't safely hold a `uint256`.

Are we sure this was returning a `number` before? Even for `web3@0.20.7` that would be weird. Maybe it was assigned the wrong type.

If this is a real breaking change, we should be sure to highlight this in the changelog